cloud_cleaner/resources/server.py

```diff
@@ -111,6 +127,73 @@ def __right_name(self, target):
                self.__name.match(target.name)
 
     def __right_age(self, target):
-        system_age = datetime.strptime(target.created, DATE_FORMAT)
+        system_age = datetime.strptime(target.launched_at, DATE_FORMAT)
         system_age = system_age.replace(tzinfo=utc)
         return self._now > (system_age + self._interval)
+
+    def __set_flagged(self):
+        """
+        Processes the txt file associated with the cloud to put all flagged
+        servers into the __flagged field.
+
+        :return: None
+        """
+        self.__flagged = []
+        conn = self._get_conn()
+        # List of server names that have been flagged to possibly be deleted
+        delete_list = []
+        # Open reader for the flagged file
+        reader = open(conn.name + "_flagged", 'r+')
```

Comment:
If we go with state-keeping, then

1) This value should be configurable, so that the filename being written is not hard-coded
2) Use a context manager for opening files to avoid having unclosed files